src/Html2Text.php

```diff
@@ -30,21 +47,38 @@ class Html2Text {
 	 * </ul>
 	 *
 	 * @param string html the input HTML
+	 * @param array An array of options of the Html2Text::OPT_* variety
```

Comment:
Should start with `@param array options An array ...` instead. Variable name is missing.